app/views/shared/_modal_state_change.html.haml

```diff
@@ -1,4 +1,5 @@
 - p1 ||= nil
+- p2 ||= nil
```

Comment:
shared/model_state_change has two incoming parameters. 
p1 is an input-field and p2 is a checkbox.  A shared view with general fields. 
P1 may represent different input fields and P2 can be used for any checkbox.